ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-dhcp-snooping.j2

```diff
@@ -0,0 +1,38 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - ip dhcp snooping #}
+{% if ip_dhcp_snooping is defined and ip_dhcp_snooping is not none %}
+
+## IP DHCP Snooping
+
+{%     if ip_dhcp_snooping.enabled is arista.avd.defined(true) %}
```

Comment:
```suggestion
{%     if ip_dhcp_snooping.enabled is arista.avd.defined(true) %}

```

Reply:
Maybe let this `if` surround the whole doc section, so we don't even print the header if it is not enabled. You can just edit the `if` in line 7 to check for `enabled` `true` directly.

Reply:
@ClausHolbechArista Sorry for that inconvenience.
I will go through your comments and try my best.

Reply:
No worries at all. Fix what you can, and I can fix the rest if anything.